src/components/AccountContainer.js

```diff
@@ -13,6 +13,7 @@ class AccountContainer extends Component {
 
     this.state = {
       searchTerm: '',
+      filteredTrans: [],
```

Comment:
We discussed this in the review, but it's probably better not to store two pieces of state. We should be able to do that filtering on the fly when the component goes to re-render.